treeherder/log_parser/artifactbuildercollection.py

```diff
@@ -83,12 +83,8 @@ def __init__(self, url, builders=None):
 
     def get_log_handle(self, url):
         """Hook to get a handle to the log with this url"""
-        req = urllib2.Request(url)
-        req.add_header('User-Agent', settings.TREEHERDER_USER_AGENT)
-        return urllib2.urlopen(
-           req,
-           timeout=settings.REQUESTS_TIMEOUT
-        )
+        headers = {'User-Agent': settings.TREEHERDER_USER_AGENT}
+        return requests.get(url, headers=headers, timeout=settings.REQUESTS_TIMEOUT)
```

Comment:
So unfortunately this change isn't as straight forwards as the others, since we need to get requests to stream us the content, which is then used below. For ideas, see:
http://docs.python-requests.org/en/latest/user/quickstart/#raw-response-content

Also it's likely we can make use of the wrapper `make_request()` in treeherder.etl.common, to avoid repetition :-)